apps/docs/src/content/reference/extras/use-fbo.mdx

```diff
@@ -0,0 +1,28 @@
+---
+order: 14.55
+category: '@threlte/extras'
+sourcePath: 'packages/extras/src/lib/hooks/useCursor.ts'
```

Comment:
howdy! well done with porting this from drei!

small thing, this link could use an update :smile:

Reply:
Ha, missed that one. It's been corrected!